src/SecureElementReader.App/Views/Titlebars/WindowsTitleBar.axaml.cs

```diff
@@ -0,0 +1,103 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+using Avalonia.Media;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace SecureElementReader.App.Views.Titlebars
+{
+    public partial class WindowsTitleBar : UserControl
+    {
+        private Button minimizeButton;
+        private Button closeButton;
+        private DockPanel titleBar;
+        private DockPanel titleBarBackground;
+        private TextBlock systemChromeTitle;
+        private NativeMenuBar seamlessMenuBar;
+        private NativeMenuBar defaultMenuBar;
+
+        public static readonly StyledProperty<bool> IsSeamlessProperty =
+        AvaloniaProperty.Register<WindowsTitleBar, bool>(nameof(IsSeamless));
+
+        public bool IsSeamless
+        {
+            get { return GetValue(IsSeamlessProperty); }
+            set
+            {
+                SetValue(IsSeamlessProperty, value);
+                if (titleBarBackground != null &&
+                    systemChromeTitle != null &&
+                    seamlessMenuBar != null &&
+                    defaultMenuBar != null)
+                {
+                    titleBarBackground.IsVisible = !IsSeamless;
+                    systemChromeTitle.IsVisible = !IsSeamless;
+                    seamlessMenuBar.IsVisible = !IsSeamless;
```

Comment:
If I remember, this should be `seamlessMenuBar.IsVisible = IsSeamless` and not negation

Reply:
<img width="636" alt="image" src="https://user-images.githubusercontent.com/350314/199412215-3e62ea89-0bc4-4be5-9370-935fb17d17d5.png">